### DISCUSSION TITLE
'TV Channel' style mode which just plays whatever live session there is in a single window

### DISCUSSION THREAD

**Discussion by double-u-a** [f1multiviewer/issue-tracker#407] [Feature requests]
Something to emulate a TV channel experience that auto replaces the current stream with the next live stream- so it can be left on in the background without having to touch it. A placeholder image could just show the schedule/session results etc if nothing live is on.

(hopefully not a duplicate but I couldn't see anything suggesting this after searching)

**Comment by jeppeolesen** [comment]
Redirecting this to #331
